src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/azure/AzureProviderNetworkingUtils.java

```diff
@@ -208,16 +201,16 @@
                                                                                              virtualNetwork.name() +
                                                                                              "'"))
                                                      .name())
-                           .withPrimaryPrivateIpAddressDynamic()
+                           .withPrimaryPrivateIPAddressDynamic()
                            .withExistingNetworkSecurityGroup(networkSecurityGroup)
-                           .withExistingPrimaryPublicIpAddress(publicIpAddress);
+                           .withExistingPrimaryPublicIPAddress(publicIpAddress);
     }
 
     public Creatable<NetworkInterface> prepareNetworkInterface(Azure azureService, Region region,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Method has 10 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00107)